Tzkt.Api/Websocket/Processors/OperationsProcessor.cs

```diff
@@ -325,17 +338,18 @@ void AddRange(HashSet<string> subs, IEnumerable<Operation> operations)
                     if (transactionsSub.All != null)
                         AddRange(transactionsSub.All, transactions.Result);
 
-                    if (transactionsSub.Addresses != null)
+                    if (transactionsSub.Addresses != null || EntrypointsSubs.Count > 0)
                         foreach (var op in transactions.Result)
                         {
-                            if (op.Initiator != null && transactionsSub.Addresses.TryGetValue(op.Initiator.Address, out var initiatorSubs))
-                                Add(initiatorSubs, op);
+                            AddTransaction(transactionsSub, op);
 
-                            if (transactionsSub.Addresses.TryGetValue(op.Sender.Address, out var senderSubs))
-                                Add(senderSubs, op);
+                            if (EntrypointsSubs.TryGetValue(op.Parameter?.Entrypoint, out var entrypointSubs))
+                            {
+                                if (entrypointSubs.All != null)
+                                    Add(entrypointSubs.All, op);
 
-                            if (op.Target != null && transactionsSub.Addresses.TryGetValue(op.Target.Address, out var targetSubs))
-                                Add(targetSubs, op);
+                                AddTransaction(entrypointSubs, op);
+                            }
```

Comment:
Since `transactionsSub` and `EntrypointsSubs` are independent and `transactionsSub` can be `null` while `EntrypointsSubs` not, we should separate them:
```cs
if (transactions.Result.Any())
{
    if (transactionsSub != null)
    {
        if (transactionsSub.All != null)
            AddRange(transactionsSub.All, transactions.Result);

        if (transactionsSub.Addresses != null)
            foreach (var op in transactions.Result)
                AddTransaction(transactionsSub, op);
    }
    if (EntrypointsSubs != null)
    {
        foreach (var op in transactions.Result.Where(x => x.Parameter != null))
        {
            if (EntrypointsSubs.TryGetValue(op.Parameter.Entrypoint, out var entrypointSubs))
            {
                if (entrypointSubs.All != null)
                    Add(entrypointSubs.All, op);

                if (entrypointSubs.Addresses != null)
                    AddTransaction(entrypointSubs, op);
            }
        }
    }
}
```